### PR TITLE
log based simulation ws for voice interaction sample app

### PR DESCRIPTION
**Pull request by xabxx**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: log_based_simulation_ws/src/person_detection_log_based_simulation/script/regression_test.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        cancel_job()
+    else:
+        rospy.loginfo("Test timed out, job already cancelled")
+        add_tags([Tag(key="status", value="timeout job already cancelled")])
+
+
+def run(clock_timeout):
+    """
+        Run the test. Connects to the /robomaker services for tagging and cancelling.
+        Then subscribes to check for any "I see" messages. Will timeout if /clock
```
Comment (timrobotson) [inline log_based_simulation_ws/src/person_detection_log_based_simulation/script/regression_test.py]: I presume it's checking for audio move messages, not "I see", there are no people in this world

File: log_based_simulation_ws/src/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# AWS RoboMaker Sample Application - Voice Interaction Log Based Simulation
+This application demonstrates the use of Amazon Rekognition to recognize people's faces using ROS bag files in simulation.
```
Comment (timrobotson) [inline log_based_simulation_ws/src/README.md]: Using Lex + Polly

File: log_based_simulation_ws/src/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+use `play_unpaused.launch` as the launch file in your AWS RoboMaker simulation job. This means that you can see the result from your simulation faster.
+
+## Interacting via AWS RoboMaker
+To inspect and interact with the job, start playback as paused. Still follow the [usage](#Usage) to setup resources but, instead, use `play_paused.launch` as the launch file. You may then interact via AWS RoboMaker console. 
+
+### Pause/Unpause via AWS RoboMaker Terminal and the ROS CLI
+1. In AWS RoboMaker console, open the Terminal GUI tool for your job.
+1. To unpause log playback, run `rosservice call /rosbag_play/pause_playback '{data: False}'`
+1. This will start the clock. To see, run `rostopic echo /clock`. 
+1. To see if people are recognized, run `rostopic echo /recoknized_people` and you should see `"I see xxx"` printed.
```
Comment (timrobotson) [inline log_based_simulation_ws/src/README.md]: Do these rosbags contain person detection data or voice data?

File: log_based_simulation_ws/.rosinstall [hunk 4]
```diff
@@ -0,0 +1 @@
+- git: {local-name: src/aws-robomaker-simulation-ros-pkgs, uri: 'https://github.com/aws-robotics/aws-robomaker-simulation-ros-pkgs.git', version: master}
```
Comment (timrobotson) [inline log_based_simulation_ws/.rosinstall:1]: Should this also contain bookstore_world like the other simulation_ws? (https://github.com/aws-robotics/aws-robomaker-sample-application-voiceinteraction/blob/master/simulation_ws/.rosinstall#L1)

File: log_based_simulation_ws/src/person_detection_log_based_simulation/script/regression_test.py (hunk 1 above)
Comment (xabxx) [reply log_based_simulation_ws/src/person_detection_log_based_simulation/script/regression_test.py]: good catch! revised.

File: log_based_simulation_ws/src/README.md (hunk 2 above)
Comment (xabxx) [reply log_based_simulation_ws/src/README.md]: yep.

File: log_based_simulation_ws/src/README.md (hunk 3 above)
Comment (xabxx) [reply log_based_simulation_ws/src/README.md]: no. updated readme.

File: log_based_simulation_ws/.rosinstall (hunk 4 above)
Comment (xabxx) [reply log_based_simulation_ws/.rosinstall:1]: We don't need to simulate the world as the logs will give input for testing the robot app.

**Review by ryanewel — APPROVED** [review]

**Review by timrobotson — APPROVED** [review]

**Pull request merged**